packages/manager/apps/pci-kubernetes/src/components/region-selector/KubeRegionSelector.component.tsx

```diff
@@ -1,40 +1,58 @@
 import {
+  PCICommonContext,
   RegionSelector,
   RegionSelectorProps,
+  TRegion,
+  usePCICommonContextFactory,
   useProductAvailability,
+  useProjectRegions,
 } from '@ovh-ux/manager-pci-common';
 import { OsdsSpinner } from '@ovhcloud/ods-components/react';
 import { ODS_SPINNER_SIZE } from '@ovhcloud/ods-components';
+import { useMemo } from 'react';
 
 export interface KubeRegionSelectorProps {
   projectId: string;
   onSelectRegion: RegionSelectorProps['onSelectRegion'];
 }
 
+const isRegionWith3AZ = (regions: TRegion[]) =>
```

Comment:
```suggestion
const areRegionsWith3AZ = (regions: TRegion[]) =>
```
Also I think it's better to have a is3AZRegion that only tests one region and put the some inside the component